api/user.go

```diff
@@ -68,10 +68,14 @@ type UpdateUserRequest struct {
 func (r UpdateUserRequest) ValidationRules() []validate.ValidationRule {
 	return []validate.ValidationRule{
 		validate.Required("id", r.ID),
-		validate.Required("password", r.Password),
 	}
 }
 
+type UpdateUserResponse struct {
+	User
+	OneTimePassword string `json:"oneTimePassword,omitempty"`
```

Comment:
I decided to omit the client changes to maintain forward compatibility. Otherwise the new CLI version will break when used against old server versions. The response is included so we can perform a seamless update in the future.